### PR TITLE
[UI] [Connection Wizard] Add 'Configuration Done' step 

### PR DESCRIPTION
**Pull request by humblenginr**
**Description**

https://user-images.githubusercontent.com/75248557/131214243-c40ddd46-27da-4767-b42d-465ffc143e3b.mov

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Task linked: [CU-7qag2z Final step after completion](https://app.clickup.com/t/7qag2z)

**Comment by warunicorn19** [comment]
Oooo, nice 😄

**Comment by leecalcote** [comment]
Lgtm

File: ui/components/ConnectionWizard/Screens/CompletionScreen.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+const useStyles = makeStyles(() => ({
+  container : {
+    position : 'relative',
+    height : '30rem',
+    textAlign : 'center',
+    marginTop : "3rem"
+  },
+  checkCircleIcon : {
+    color : '#00B39F',
```
Comment (leecalcote) [inline ui/components/ConnectionWizard/Screens/CompletionScreen.js:14]: It'll be good to centrally reference the theme is available.￼ // @Tanuj22

**Review by leecalcote — APPROVED** [review]

**Pull request merged**